crates/swc_plugin_runner/src/lib.rs

```diff
@@ -96,14 +97,42 @@ fn load_plugin(plugin_path: &Path, cache: &mut Option<PluginCache>) -> Result<In
 
     return match module {
         Ok(module) => {
-            let import_object = imports! {};
+            let memory = Memory::new(&wasmer_store, MemoryType::new(1, None, false))?;
```

Comment:
What does this code do?

Reply:
This creates memory `instance` for the given wasm instance we runs. then expose it into guest (wasm) instance via https://docs.rs/wasmer/2.0.0/wasmer/#externs.